app/models/purchase.py

```diff
@@ -57,12 +57,28 @@ def __init__(self, cart_items):
     def __repr__(self):
         return "<Order: {}>".format(self.id)
 
+    def getAllPurchases(self):
+        return Purchase.query.filty_by(order_id=self.id)
```

Comment:
`filty` should be `filter` and shouldn't you return a list of purchases a la `.all()`?